test/async-hooks/test-embedder.api.async-resource.js

```diff
@@ -18,8 +18,8 @@ assert.throws(() => new AsyncResource('invalid_trigger_id', null),
               /^RangeError: triggerAsyncId must be an unsigned integer$/);
 
 assert.strictEqual(
-    typeof new AsyncResource('default_trigger_id').triggerAsyncId(),
```

Comment:
This change wasn't needed. But when I looked for the `AsyncResource('default_trigger_id')` test, I found that it could be made more strict.